.github/workflows/pythonpackage.yml

```
@@ -0,0 +1,34 @@
+name: Python package
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.8]
```

It would be useful to test a few python versions. 3.9, 3.8, 3.7.

```suggestion
        python-version: [3.9, 3.8, 3.7]
```